docs/comunidad/charlas-y-presentaciones.md

```diff
@@ -0,0 +1,122 @@
+# **Presentaciones y vídeos**
+
+En esta sección encontrarás presentaciones y vídeos de charlas impartidas por nosotros.
+
+___
+
+![EOS Costa Rica pre practica](/static/img/charlas/Pre-practica.webp)
+
+
+
+
+## Tema: EOSCR Pre-practica UI/UX
+
+**Por:** Jorge Murillo.
+
+**Idioma:** Español.
+
+**Fecha:** 4 de Febrero, 2021.
+
+### Grabación de la charla:
+
+**[Enlace](https://us02web.zoom.us/rec/share/WWDqs5rB2r5rEdulNKUbOHcTBfiw6_ZTII8fB-UBjabJrQ3wTBvndkvnpWJupeBL.2n3yzt_rczzmKzj3)**
```

Comment:
Lets avoid using the Word Link in a link 

lets use keywords for links intead

for example : **[Video Charla UI/UX Blockchain](https://us02web.zoom.us/rec/share/WWDqs5rB2r5rEdulNKUbOHcTBfiw6_ZTII8fB-UBjabJrQ3wTBvndkvnpWJupeBL.2n3yzt_rczzmKzj3)**